SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc

```diff
@@ -1,70 +1,71 @@
-

```

Comment:
The whole file is diffed. For some reason your editor changed the linebreaks from LF (Unix) to CRLF (Windows). This needs to be reverted. Can you check how that could have happened? Apparently, the only file that is affected is this one here.